app/src/main/java/com/example/byitscover/ReviewPage.java

```diff
@@ -245,6 +245,7 @@ private void setGoodreadsInfo(View view, BookListing listing) {
         }
         catch (Exception e) {
             System.out.println(e.toString());
+            goodReadsResultPrice.setText("$" + listing.getPrice().toString());
```

Comment:
Should this be `goodReadsResultPrice.setText("Pricing Info Not Provided");`?

Reply:
Yup good catch. Forgot to change in my copy-paste